channels/views/channels.py

```diff
@@ -52,6 +57,9 @@ def get_serializer_context(self):
         """Context for the request and view"""
         return {
             'channel_api': self.request.channel_api,
+            'channels': {
```

Comment:
Can you explain what the benefit is of adding `channels` to the context?

Reply:
It's to cache `Channel` objects so they aren't fetched repeatedly. Otherwise we would need to fetch it once per field for `avatar`, `banner` and `membership_is_managed`, times once per `Channel` being shown in the list